spec/requests/create_applicant_spec.rb

```diff
@@ -30,8 +31,9 @@
       post '/v1/applicants', params: body, headers: headers
       expect(response.status).to eql(201)
       expect(response.content_type).to eql('application/json')
-      expect(response_json['data']['attributes']['name']).to eq('John Doe')
-      expect(response_json['data']['attributes']['date_of_birth']).to eq('1991-12-01')
+      expect(response_json.dig('data', 'attributes', 'first_name')).to eq('John')
```

Comment:
**[not a blocker]** 💭 Consider [this approach](https://github.com/ministryofjustice/laa-apply-for-legalaid-api/blob/master/spec/requests/legalaid_application_spec.rb#L47) as an alternative way of asserting the same content && schema without having to assert each field separately.

Reply:
is this possible on a POST request? as it is generating a new id that we dont know how to match on

Reply:
Yes, if you look at the gem in [more detail](https://github.com/chancancode/json_expressions#usage) you will find some example where we can define a more dynamic way of expecting a given value we don't know its exact content, but we know it's a valid UUID.